roles/ee_builder/README.md

```diff
@@ -192,26 +197,34 @@ This is an example for building using automated pipelines like Gitlab or Azure D
         # To reiterate, only the name variable goes here, not the namespace, that is placed in ee_registry_dest, please refer to ee_registry_dest's description for more details
       - name: custom_ee
         # Using the latest tag is best practice and should be replaced with a tested version of the container. However latest can be a good starting point to figure out which container works, then replacing latest with the version number for the tested latest container.
-        base_image: registry.redhat.io/ansible-automation-platform/ee-minimal-rhel8:latest
-        builder_image: registry.redhat.io/ansible-automation-platform-23/ansible-builder-rhel8:latest
-        bindep:
-          - python38-requests [platform:centos-8 platform:rhel-8]
-          - python38-pyyaml [platform:centos-8 platform:rhel-8]
-        python:
-          - pytz  # for schedule_rrule lookup plugin
-          - python-dateutil>=2.7.0  # schedule_rrule
-          - awxkit  # For import and export modules
-        collections:
-          - name: awx.awx
-            type: url
-            source: https://galaxy.ansible.com/download/awx-awx-21.11.0.tar.gz
-          - name: infra.controller_configuration
-          - name: infra.ah_configuration
-        prepend:
-          - RUN whoami
-          - RUN cat /etc/os-release
-        append:
-          - RUN echo This is a post-install command!
+        images:
+          base_image:
+            name: registry.redhat.io/ansible-automation-platform-21/ee-minimal-rhel8:latest
```

Comment:
This should reference the AAP 2.4 RHEL9 image IMHO.

Reply:
yes, already updated